README.md

```diff
@@ -50,7 +50,8 @@ ElasticSearch source connector allows you to move data from multiple Elasticsear
 | `apiKey`                 | [v: 6, 7, 8] Base64-encoded token for authorization; if set, overrides username/password and service token.                                                                                                                                      | `false`                                              |          |
 | `serviceToken`           | [v: 7, 8] Service token for authorization; if set, overrides username/password.                                                                                                                                                                  | `false`                                              |          |
 | `certificateFingerprint` | [v: 7, 8] SHA256 hex fingerprint given by Elasticsearch on first launch.                                                                                                                                                                         | `false`                                              |          |
-| `indexes`                  | The name of the indexes to read the data from.                                                                                                                                                                                                      | `true`                                               |          |
+| `indexes.*.sortBy`                  | The sortby field for each index to be used by elasticsearch search api.(A field must be specified for v5, v6 as it does not support sorting using the default `_seq_no`)                                                                                                                                                                                                       | `true`                                               |    `_seq_no`      |
+| `indexes.*.sortOrder`                  | The sortOrder (asc or desc) for each index to be used by elasticsearch search api.                                                                                                                                                                                                      | `true`                                               |   `asc`       |
```

Comment:
Same as above, this field shouldn't be required when providing a default value.

```suggestion
| `indexes.*.sortOrder`                  | The sortOrder (asc or desc) for each index to be used by elasticsearch search api.                                                                                                                                                                                                      | `true`                                               |   `asc`       |
```

Reply:
updated.